package.json

```
@@ -4,7 +4,8 @@
     "description": "Posthog-js allows you to automatically capture usage and send events to PostHog.",
     "repository": "https://github.com/PostHog/posthog-js",
     "author": "hey@posthog.com",
-    "license": "MIT",
+    "license": "Apache-2.0",
```

I've checked https://docs.npmjs.com/cli/v11/configuring-npm/package-json#license and in our case where we have a multilicense we should do this

```suggestion
    "license": "SEE LICENSE IN LICENSE",
```